scripts/compact.js

```diff
@@ -195,6 +199,34 @@ var altTracker = `<table width="100%">
         <br>
         <p data-chapter-key-count="16">0/120</p>
     </div>
+    <div id="DungeonEntranceRow" class="compact-element">
+        <img id="Unknown" data-state="0" class="boss" src="images/bosses/unknown.png">
+        <p class="boss-chapter">Ch.1</p>

```

Comment:
The chapter text here appears overlaid with the image but is not clickable. Might consider moving click handler to containing div.